### ISSUE TITLE
Huge issue with NPC gear upgrade window randomly opening AH window

### ISSUE THREAD

**Issue by Rezy942** [nezroy/DeModal#30]
Example:
Happens when I open upgrade window...
I use Auctionator if it matters, but I doubt it.
Tested extensively. Issue only happens with Demodal

Please fix :)

**Comment by Chromos-Def** [comment]
This happens without DeModal. I haven't used DeModal in months since no update and the AH occasionally opens on me and I have to reload UI to fix it.

**Comment by Rezy942** [comment]
Thats weird. I use it on a daily basis with 100 different addons. Never any issues until I download and enable DeModal.

**Comment by biggianteye** [comment]
I've experienced something similar but it's the map that shows up rather than the auction house. Unclear if it's exactly the same problem.

**Comment by Chromos-Def** [comment]
Yeah, I believe it is just a random bug that could possibly be exacerbated by DeModal for it to occur a lot more often or something of the sorts.

**Comment by Rezy942** [comment]
> I've experienced something similar but it's the map that shows up rather than the auction house. Unclear if it's exactly the same problem.

Yes, can confirm, sometimes the map was opening up. All while using DeModal.
